### PR TITLE
Add a query for removing coordinators

### PR DESCRIPTION
**Pull request by as51340**
Added `REMOVE COORDINATOR <ID>` query. Only followers can be removed from Raft logs. The request for removing a leader won't be accepted in Raft logs.

### CODE REVIEW

**Comment by as51340** [comment]
### Tracking
- [ ] **[Link to Epic/Issue]**


### Standard development
- [ ] Update unit/E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch this branch


### CI Testing Labels
- [ ] Select the appropriate CI test labels _(CI -build=**build-name** -test=**test-suite**)_


### Documentation checklist
- [ ] Add the documentation label
- [ ] Add the bug / feature label
- [ ] Add the milestone for which this feature is intended
    - If not known, set for a later milestone
- [ ] Write a release note, including added/changed clauses
    - **[ Release note text ]**
- [ ] **[ Documentation PR link memgraph/documentation#XXXX ]**
    - [ ] Is back linked to this development PR
- [ ] **[ Tag someone from docs team ]**

File: src/coordination/raft_state.cpp [hunk 1]
```diff
@@ -259,6 +259,40 @@ auto RaftState::GetCoordinatorToCoordinatorConfigs() const -> std::vector<Coordi
...
+  } else {
+    throw RaftRemoveServerException(
+        "Failed to accept request for removing coordinator {} from the cluster with the error code {}", coordinator_id,
+        int(cmd_result->get_result_code()));
+  }
+
+  // Waiting for server to join
+  constexpr int max_tries{10};
+  auto maybe_stop = utils::ResettableCounter<max_tries>();
+  constexpr int waiting_period{200};
```
Comment (andrejtonev) [inline src/coordination/raft_state.cpp:278]: just define the milliseconds right here, not an int that then gets converted to milliseconds

File: src/coordination/raft_state.cpp [hunk 2]
```diff
@@ -282,7 +316,7 @@ auto RaftState::AddCoordinatorInstance(CoordinatorToCoordinatorConfig const &con
   while (!maybe_stop()) {
     std::this_thread::sleep_for(std::chrono::milliseconds(waiting_period));
     const auto server_config = raft_server_->get_srv_config(static_cast<nuraft::int32>(config.coordinator_id));
-    if (server_config) {
+    if (server_config->get_id() == config.coordinator_id) {
```
Comment (andrejtonev) [inline src/coordination/raft_state.cpp:319]: do we need to check this? aren't we searching via the id

**Review by andrejtonev — APPROVED** [review]

File: src/coordination/raft_state.cpp (hunk 2 above)
Comment (as51340) [reply src/coordination/raft_state.cpp:319]: No you are right

**Pull request merged**